### ISSUE TITLE
[Bug]: Milvus hangs at flush after deleting rootcoord pod kill chaos

### ISSUE THREAD

**Issue by zhuwenxing** [milvus-io/milvus#12339]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: master-20211127-2c484c7
- Deployment mode(standalone or cluster): cluster
- SDK version(e.g. pymilvus v2.0.0rc2): 2.0.0rc9.dev16
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Milvus hangs at flush when all pods recovered to ready status after deleting proxy pod kill chaos

### Expected Behavior

All functions work well

### Steps To Reproduce

```markdown
cd `tests/python_client/chaos`

modify params:
pod="rootcoord"
chaos_type="pod_failure"
chaos_task="chaos-test"

run script `bash chaos_test.sh`
```

### Anything else?

[test-rootcoord-pod-kill-11-29-15-21-00.zip](https://github.com/milvus-io/milvus/files/7616273/test-rootcoord-pod-kill-11-29-15-21-00.zip)

**Comment by sunby** [comment]
Fixed in #12303 , please check it @zhuwenxing 
/assign @zhuwenxing 
/unassign @sunby

**Comment by zhuwenxing** [comment]
Verified and passed in version `master-20211130-074c36d`
